### PR TITLE
CCDA server fixups

### PR DESCRIPTION
**Pull request by sjpadgett**
- add some window escaping in dispatch
- sanity check if server is running. ms count!
- add lodash dependency to server

### CODE REVIEW

**Comment by sjpadgett** [comment]
Will leave open for a bit. CCD is being tested by N.Carolina HIE for compliance so, though minimum requirements required, will give some insights.

**Comment by bradymiller** [comment]
go CCDA!!

tenor (53)

File: interface/main/backup.php [hunk 1]
```diff
@@ -814,19 +814,35 @@ function export_submit(step) {
...
+                // Beware and keep in mind that Windows requires double quotes around arguments.
+                if (IS_WINDOWS) {
+                    # windows will place the quotes in the outputted code if they are there. we removed them here.
+                    $cmd .= " echo 'DELETE FROM layout_options WHERE form_id = \"" . add_escape_custom($layoutid) . "\";' >> " . escapeshellarg($EXPORT_FILE) . " & ";
+                } else {
+                    $cmd .= "echo 'DELETE FROM layout_options WHERE form_id = \"" . add_escape_custom($layoutid) . "\";' >> " . escapeshellarg($EXPORT_FILE) . ";";
+                }
+                if (IS_WINDOWS) {
+                    # windows will place the quotes in the outputted code if they are there. we removed them here.
+                    $cmd .= "echo 'DELETE FROM layout_group_properties WHERE grp_form_id = \"" . add_escape_custom($layoutid) . "\";' >> " . escapeshellarg($EXPORT_FILE) . " &;";
```
Comment (sunsetsystems) [inline interface/main/backup.php:826]: Is " &;" at the end intentional here, or should it be " & " ?

Comment (sjpadgett) [reply interface/main/backup.php:826]: Nope, correct as is. Thanks for looking at though.

Comment (sjpadgett) [reply interface/main/backup.php:826]: I do have to figure out where new required dependency layout.inc.php came from and whether i'll be able to bring back for patch 2.

Comment (sunsetsystems) [reply interface/main/backup.php:826]: It's needed for CSV export of layouts, which is not in 6.0.0.

Comment (sjpadgett) [reply interface/main/backup.php:826]: Do we want it in 6 or leave it be as current backup for 6.0 works fine? I can bring back if wanted

Comment (sunsetsystems) [reply interface/main/backup.php:826]: I would not add it to 6... most new features go into master only.

**Comment by sjpadgett** [comment]
reverted backup commit so I can bring this PR back for patch 2. will do PR for  backup...

**Pull request merged**